cpp/Ice/greeter/README.md

```diff
@@ -2,12 +2,37 @@
 
 The Greeter demo illustrates how to send a request and wait for the response.
 
+To build the demo run:
+
+```shell
+cmake -B build
+cmake --build build --config Release
+```
+
 To run the demo, first start the server:
+
+**Linux/macOS:**
+
+```shell
+./build/server
 ```
-server
+
+**Windows:**
+
+```shell
+./build/Release/server
```

Comment:
Should this be:
```shell
build\Release\server
```
?

Reply:
Yes.

Reply:
Note that this pathing is really generator specific and not specific. But since we're showing the defaults which are Make and "Visual Studio" I think it's fine to phrase like this.